### ISSUE TITLE
TypeError: utlis.Promise.all is not a function

### ISSUE THREAD

**Issue by vickydadhich** [calvinmetcalf/crypto-pouch#69]
Getting below error after importing crypto-pouch plugin.
`Error: Uncaught (in promise): TypeError: utils.Promise.all is not a function
TypeError: utils.Promise.all is not a function
    at index.js:105
    at t.invoke (polyfills.js:3)
    at Object.onInvoke (core.js:4626)
    at t.invoke (polyfills.js:3)
    at r.run (polyfills.js:3)
    at polyfills.js:3
    at t.invokeTask (polyfills.js:3)
    at Object.onInvokeTask (core.js:4617)
    at t.invokeTask (polyfills.js:3)
    at r.runTask (polyfills.js:3)
    at index.js:105
    at t.invoke (polyfills.js:3)
    at Object.onInvoke (core.js:4626)
    at t.invoke (polyfills.js:3)
    at r.run (polyfills.js:3)
    at polyfills.js:3
    at t.invokeTask (polyfills.js:3)
    at Object.onInvokeTask (core.js:4617)
    at t.invokeTask (polyfills.js:3)
    at r.runTask (polyfills.js:3)
    at c (polyfills.js:3)
    at c (polyfills.js:3)
    at polyfills.js:3
    at t.invokeTask (polyfills.js:3)
    at Object.onInvokeTask (core.js:4617)
    at t.invokeTask (polyfills.js:3)
    at r.runTask (polyfills.js:3)
    at o (polyfills.js:3)
    at e.invokeTask [as invoke] (polyfills.js:3)
    at p (polyfills.js:2)`
So basically I am using this plugin in an Ionic 3 project and injecting in custom database service. whenever an object of db is created and db.crypto() function is called, it throws above error.

**Comment by garbados** [comment]
Howdy! The project has just released 4.0, which included a lot of significant changed. If it's still relevant to your work, I invite you to try again. Let me know how it goes!

**Comment by garbados** [comment]
I'm going to close this issue for now since it's been several years, but I can re-open it at your request :)
